scripts/build_clone_docs.sh

```diff
@@ -11,20 +11,31 @@ CUR_DIR="$(cd $(dirname $0) && pwd)"
 
 git_clone_doc_tags() {
 
-    #get all tags from the repo
-    TAGS=$(git tag -l --sort=-v:refname)
-    #get the latest tag from repo
-    LATEST_TAG=$(git tag -l --sort=v:refname | tail -1)    
+    #get all tags we want to clone from the repo
+    TAGS_TO_CLONE=$($CUR_DIR/tagScript.js $(git tag -l)) 
+
+    #get rid of characters we dont need
+    TAGS=${TAGS_TO_CLONE//[[\',\]]/}
+    
+    #sort the array and set the tag with the greatest value as the latest tag
+    LATEST_TAG=($(for each in ${TAGS[@]}; do echo $each; done | sort | tail -1))
+
+    #parse out only the major.minor values for the latest tag
+    LATEST_MAJOR_MINOR=${LATEST_TAG%.*}
```

Comment:
I would try to delegate this stuff to the tagScript.js. Whatever output you need should be handled there. It will reduce the size and complexity of this script